ci/setup-build-test-gdk.ps1

```diff
@@ -19,20 +19,18 @@ class TestProjectTarget {
         $this.test_repo_url = $test_repo_url
         $this.test_repo_relative_uproject_path = $test_repo_relative_uproject_path
         $this.test_project_name = $test_project_name
-        $this.test_gyms_version_path = $test_gyms_version_path
-        $this.test_env_override = $test_env_override
         
         # Resolve the branch to run against. The order of priority is:
         # envvar > same-name branch as the branch we are currently on > UnrealGDKTestGymVersion.txt > "master".
         $testing_repo_heads = git ls-remote --heads $test_repo_url $gdk_branch
         $test_gym_version = if (Test-Path -Path $test_gyms_version_path) {[System.IO.File]::ReadAllText($test_gyms_version_path)} else {[string]::Empty}
         if (Test-Path $test_env_override) {
-            $this.test_repo_branch = $test_env_override
+            $this.test_repo_branch = (Get-Item $test_env_override).value
```

Comment:
Why is this needed?

Reply:
We are now passing the environment variables by the path where they can be found. Test-Path works on the path as a string, but then we also need a corresponding function to actually retrieve the environment variable's value from that path.

Reply:
Super minor: reading the docs this seems like the nicest way to do it. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-itemproperty?view=powershell-7.1#example-4--get-the-value-name-and-data-of-a-registry-entry-in-a-registry-subkey

Reply:
![image](https://user-images.githubusercontent.com/2778498/106607583-66705800-655b-11eb-9345-0c4a06bc62c2.png)

Reply:
`Get-ItemProperty -Path HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion -Name "value"`
I think is the format

Reply:
I tried a few ways, seems tailored to registry values or something

Reply:
fair enough